config/config.properties

```diff
@@ -110,6 +110,24 @@ pravegaservice.dataLogImplementation=BOOKKEEPER
 # and fetch it back when needed.
 #pravegaservice.cacheMaxSize=17179869184
 
+# Percentage (of pravegaservice.cacheMaxSize) that defines target Local Shared Cache. The Segment Store will try to keep
+# the cache utilization at or below this value, and may apply throttling on new operations if it exceeds it.
+# Valid values: 1 to 100 (inclusive).
+# Recommended values: Between 80 and 90. It is not recommended to choose a very high value as that may leave very little
+# buffer for the Segment Store to attempt any cleaning, apply back-pressure or serve other random requests. Consequently,
+# a very low value may have negative impact on the ingestion performance.
+#pravegaservice.cacheTargetUtilizationPercent=85
+
+# Percentage (of pravegaservice.cacheMaxSize) that defines the maximum usable size of the Local Shared Cache. The Segment
+# Store will apply full throttling if reaching or exceeding this value.
+# Valid values: 1 to 100 (inclusive).
+# Recommended values: Between pravegaservice.cacheTargetUtilizationPercent + 10 and 99. Setting a value too close to
+# pravegaservice.cacheTargetUtilizationPercent will cause the Segment Store to apply maximum throttling as soon as that
+# limit is reached, which may cause it to have sudden, multi-second stalls in the ingestion pipeline. It is not recommended
+# to set this value ever to 100, since that would mean the cache is already full when max throttling kicks in, and the
+# Segment Store will likely shut down Segment Containers in order to alleviate cache pressure.
+#pravegaservice.cacheMaxUtilizationPercent=98
```

Comment:
This will throw an exception upon starting the segment store process. This is consistent with supplying other invalid (combinations of) configuration values. I believe it's best to prevent startup than altering configurations behind the scenes which could leave users (and debuggers) in the dark about what is going on.